### DISCUSSION TITLE
Shutter: (temporary) Disable software endstops and switch relays

### DISCUSSION THREAD

**Discussion by dakky** [arendst/Tasmota#14917] [Q&A] [answered]
Hello

I have shelly 2.5 with latest tasmota running in order to control my shutter.

There are 2 problems at the moment:

## calibration
The documentation tells me to start the calibration with a completely closed shutter. 

The problem: I can't close it completely, as tasmota stops the movement. So basically i need a way, to manually close the shutter by ignoring the software endstop. 

## switching the buttons/relays

At first i had wired 
* relay1  == down 
* relay2 == up. 

This resulted in `shutterclose` opening the shutter and vice versa.
this problem was solved by switching the wires, but I am curious, why the "software" way didn't work:
```
# configure buttons (before fixing the wiring)
ShutterButton1 1 down 1
ShutterButton1 2 up 1
```

my tasmota setup:
```
# create new tempalte
Template {"NAME":"Shelly 2.5","GPIO":[56,0,17,0,21,83,0,0,6,82,5,22,156],"FLAG":2,"BASE":18}
# activate new template
Module 0
# enable shutter 
SetOption80 
# select shutter mode
ShutterMode 1
# just in case: activate interlock
Interlock 1
# configure buttons (after fixing the wiring)
ShutterButton1 1 up 1
ShutterButton1 2 down 1
```

**Comment by stefanbode** [comment, marked as answer]
SETOPTION80 0 releases all relays and disable shutter. No reboot required
